### PR TITLE
SPCTR-2576 Animations - Fix UI for the Animation Type Select control, upgrade to Spectra Range Controls, add support to child blocks, remove support within tabs

### PR DESCRIPTION
**Pull request by aapatre**
### Description
- Sleek, compact design for Animation type select dropdown
- Categorization of options within the drop down
- Style-guide based changes for above
- Upgraded range controls to Spectra's Range control, fixed CSS

### Screenshots

image

image

### Types of changes

- UX improvements

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by adi3890 — APPROVED** [review]

File: src/blocks/extensions/editor.lazy.scss [hunk 1]
```diff
@@ -102,3 +104,92 @@
...
+		box-shadow: none;
+		border: none;
+	}
+
+	.uagb-animation-type-select__control {
+		cursor: pointer;
+		border: 1px solid $spectra-color-border;
+		box-shadow: none;
+		height: 30px;
+		min-height: 30px;
```
Comment (bsf-zanev) [inline src/blocks/extensions/editor.lazy.scss:129]: Could we use the CSS variable `$spectra-control-input-height` for height and min height?

File: src/blocks/extensions/editor.lazy.scss [hunk 2]
```diff
@@ -102,3 +104,92 @@
...
+		border-radius: 3px;
```
Comment (bsf-zanev) [inline src/blocks/extensions/editor.lazy.scss:130]: Variable `$spectra-control-border-radius`

File: src/blocks/extensions/editor.lazy.scss [hunk 3]
```diff
@@ -102,3 +104,92 @@
...
+		svg {
+			height: 14px;
+			width: 14px;
+		}
+	}
+
+	.uagb-animation-type-select__value-container {
+
+		top: 0;
+		height: 30px;
```
Comment (bsf-zanev) [inline src/blocks/extensions/editor.lazy.scss:151]: Variable `$spectra-control-input-height` here if possible.

**Review by bsf-zanev — COMMENTED** [review]
Could you implement the following CSS variables?

All done cc: @bsf-zanev

**Review by bsf-zanev — APPROVED** [review]

**Comment by aapatre** [comment]
@aadi704 @rahulvarma722 this CSS lint is conflicting with `npm run lint-scripts:fix`

image

cc: @RajkiranBagal

**Pull request merged**